### PR TITLE
Display weekly download numbers

### PR DESCRIPTION
**Pull request by marikaner**
Add a script to log weekly download numbers.

### CODE REVIEW

File: scripts/npm-stats.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { JSDOM } from 'jsdom';
+
+async function getStatsForPackage(packageName) {
+  const { data } = await axios.get(
+    `https://www.npmjs.com/package/${packageName}?activeTab=versions`
+  );
+
+  const dom = new JSDOM(data);
+  const ul = dom.window.document.querySelector(
+    '.a0dff0b1 ~ .c495d29d ~ .a0dff0b1 ~ .c495d29d'
```
Comment (florian-richter) [inline scripts/npm-stats.ts:11]:
```suggestion
    '#tabpanel-versions > * > ul:last-child'
```

I find this a bit more speaking (and hopefully less likely to break)

File: scripts/npm-stats.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const res: any[] = [];
+
+  for (let i = 1; i < versionLines.length; i++) {
+    const line = versionLines.item(i);
+    const [version, _, downloads] = line.children;
+    res.push({
+      version: version.textContent,
+      downloads: parseInt(downloads.textContent.replace(/,/g, ''))
+    });
+  }
```
Comment (florian-richter) [inline scripts/npm-stats.ts:23]:
```suggestion
 Array.from(versionLines).map(line => line.children).map(([version,_,dl]) => ({ version: version.textContent, downloads: parseInt(dl.textContent.replace(/,/g, '')) }))
```
Maybe some cleanup needed but the `.map()` chain seems nicer to me

File: scripts/npm-stats.ts (hunk 1 above)
Comment (jjtang1985) [inline scripts/npm-stats.ts:11]: 🤔 This might need further improvement as a follow-up

File: scripts/npm-stats.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      ...(await stats),
+      [packageName]: await getStatsForPackage(packageName)
+    }),
+    {}
+  );
+}
+
+async function main() {
+  // eslint-disable-next-line no-console
+  console.log(await getAllStats());
```
Comment (florian-richter) [inline scripts/npm-stats.ts:50]: Could we create a GH repo where we upload the stats every day? Maybe using this: https://github.com/octokit/octokit.js

**Review by florian-richter — APPROVED** [review]
Works! But some improvement suggestions:

**Pull request merged**